expenses/bot.py

```diff
@@ -132,16 +133,27 @@ def record(update, context):
 
 
 def report(update, context):
+    conn = sqlite3.connect('db.sql')
```

Comment:
Let's use the existing `db` to connect to the database.

You can use https://github.com/theSage21/expenses/blob/4cde74bd5f7b83c88daee8c543c1bb8a2a7c8931/expenses/bot.py#L116-L126 to access the database instead of using sqlite3 directly.